apps/studio/components/interfaces/Storage/AnalyticBucketDetails/DecryptedReadOnlyInput.tsx

```diff
@@ -41,52 +42,57 @@ export const DecryptedReadOnlyInput = ({
     : value
 
   return (
-    <Input
-      readOnly
-      // If the value is secure, allow copying to clipboard if the value is revealed. Otherwise, always allow copying
-      copy={!secureEntry || (!isDecryptedValueLoading && showHidden)}
-      disabled
```

Comment:
We don’t need `disabled` if `readOnly` is the intent. That’s something I’ve cleaned up stylistically in https://github.com/supabase/supabase/pull/39509